lug/config.yaml

```diff
@@ -14,10 +14,162 @@ dummy:
     retry_interval: 60
 
 repos:
-  - type: shell_script
+  #archlinux
+  - type: shell_script 
     script: /worker-script/rsync.sh
-    source: rsync://archive.ubuntu.com/ubuntu
+    source: rsync://rsync.archlinux.org/ftp_tier1/
     interval: 3600
+    path: /srv/archlinux
+    name: archlinux
+    <<: *oneshot_common
+  #centos
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://msync.centos.org/CentOS/
+    interval: 21600
```

Comment:
Please try to interleave the interval. For all `21600` entries, +- a random value < 3600 for them. Otherwise lug will be flooded with pending sync tasks.

Reply:
fixed